templates/main.php

```diff
@@ -84,20 +84,16 @@
 		</div>
 
 		<?php if(!$_['isPublic']): ?>
-		<div ng-controller="CalendarListController" id="calendarlistcontainer" ng-cloak>
-			<div id="scrollable">
```

Comment:
@skjnldsv You can't remove `#scrollable` here, it served a very important purpose:

![kapture 2017-12-19 at 15 20 34](https://user-images.githubusercontent.com/1250540/34161212-3042a418-e4d0-11e7-94f3-14cefbb7e00a.gif)

The lower part of the list is not hidden behind the settings button